system/jlib/jbuff.cpp

```diff
@@ -66,7 +66,7 @@ constexpr unsigned BUFF_DETACH_GRANULARITY=16;
 #ifdef _DEBUG
 #define     CHECKREADPOS(len)  assertex(readPos+(len)<=length())
 #else
-#define     CHECKREADPOS(len)  
+#define     CHECKREADPOS(len)  assertex(readPos+(len)<=length())
```

Comment:
any concerns about the extra few cycles per ::read**( call ?

As they're the same, I'd be inclined to get rid of the #ifdef _DEBUG